### ISSUE TITLE
IntelliJ 2024 Code Completion broken

### ISSUE THREAD

**Issue by YunaBraska** [YunaBraska/github-workflow-plugin#57]
The new Version of IntelliJ brings also some GitHub workflow features which now overrides every code completion and shows `no suggestions` The variables are highlighted but not completed by code completion anymore.

**Comment by LecrisUT** [comment]
Not sure I can replicate this issue, at least not after #63

**Comment by erichiller** [comment]
I just upgraded to 2024.1 and found code completions broken as well. Let me know if there's any information I can provide, but I imagine everyone is experiencing this. Hopefully there is a decent fix because I tried their "native" support and it is vastly inferior to your plugin!

**Comment by YunaBraska** [comment]
@erichiller, so you think this plugin is not needed anymore?

**Comment by LecrisUT** [comment]
> @erichiller, so you think this plugin is not needed anymore?

They were saying the opposite :P

@erichiller I am curious, how is the code completion in [this version](https://github.com/YunaBraska/github-workflow-plugin/actions/runs/8665866511?pr=63)? (There's `Github-Workflow-Plugin-3.2.1` archive there and you should be able to install manually with `install from disk` option in `Plugins`. Not sure if the version needs to be consistent) On my limited testing, it seems some completion works, but I haven't written any tests or dug deeper to verify what I did in that PR makes sense.

**Comment by erichiller** [comment]
> @erichiller, so you think this plugin is not needed anymore?

No! This plugin is far superior / much better then the functionality included in default Jetbrains (Rider in my case).

@LecrisUT I will check as soon as possible and get back to you

**Comment by erichiller** [comment]
I went to try the workflow built version you linked to - I went to my IDE and it required a reboot for updating some other plugin. I did that and when it came back up all the completions in github actions worked as they should. I guess this is solved for me. Sorry to create a non-issue, I'll you know if it re-occurs.

**Comment by YunaBraska** [comment]
For me the issue still exist, telling me `no suggestion`
